staging/src/github.com/kubeedge/mapper-framework/README.md

```diff
@@ -43,5 +45,3 @@ mapper
 └── Makefile
```

Comment:
The file tree can be moved to docs instead of in readme, and we can add the link to the overall docs at here. You can open an issue to track it.

Reply:
Done. I opened it in #5268